src/UI/DatePicker.elm

```diff
@@ -0,0 +1,477 @@
+module UI.DatePicker exposing (DateEvent(..), DatePicker, Msg, datepicker, init, update)
+
+import Date exposing (Date, Interval(..), Unit(..))
+import Element exposing (Element, alignLeft, alignRight, fill, fillPortion, minimum, px, spacing, width)
+import Element.Border as Border
+import Element.Events
+import Element.Font as Font
+import Time
+import UI.Button as Button
+import UI.Icon as Icon
+import UI.Palette as Palette
+import UI.RenderConfig as RenderConfig exposing (RenderConfig)
+import UI.Size as Size
+import UI.Text as Text
+
+
+{-| The `UI.Datepicker` is a component for displaying a calendar.
+
+User must specify da today date, the date selected to construct it.
+
+
+# Building
+
+@docs DatePicker, datepicker
+
+
+# Content
+
+@docs init, update
+
+-}
+type alias Property msg =
+    { today : Date
```

Comment:
I would make today's mark optional.

Reply:
from the design we show a "." under the number of today